src/payment/strategies/paypal-commerce/paypal-commerce-credit-card-payment-strategy.ts

```diff
@@ -48,7 +52,7 @@ export default class PaypalCommerceCreditCardPaymentStrategy implements PaymentS
 
         await this._store.dispatch(this._orderActionCreator.submitOrder(order, options));
 
-        const { orderId } = await this._paypalCommerceHostedForm.submit();
+        const { orderId } = await this._paypalCommerceHostedForm.submit(this._is3dsEnabled);
```

Comment:
You can get the value directly from the state so you don't need to store a temporary variable and having to unset it later. 🍹  i.e.:

```ts
const { paymentMethods: { getPaymentMethodOrThrow } } = await this._store.dispatch(this._orderActionCreator.submitOrder(order, options));
const { orderId } = await this._paypalCommerceHostedForm.submit(getPaymentMethodOrThrow().config.is3dsEnabled);
```

Reply:
Thank you!